cwms-data-api/src/test/java/cwms/cda/api/TimeseriesControllerTestIT.java

```diff
@@ -1,709 +1,28 @@
 package cwms.cda.api;
 
-import java.io.InputStream;
-import java.nio.charset.StandardCharsets;
+import java.sql.SQLException;
 import javax.servlet.http.HttpServletResponse;
 
+
 import cwms.cda.formatters.Formats;
 import fixtures.TestAccounts;
-import io.jsonwebtoken.io.IOException;
+import io.restassured.RestAssured;
 import io.restassured.filter.log.LogDetail;
+import io.restassured.path.json.config.JsonPathConfig;
 import org.apache.commons.io.IOUtils;
-import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.JsonNode;
 
 import static io.restassured.RestAssured.given;
+import static io.restassured.config.JsonConfig.jsonConfig;
 import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 
 @Tag("integration")
 public class TimeseriesControllerTestIT extends DataApiTestIT {
-
-    private static final String OFFICE = "SPK";
-    private static final String locationId = "TsVersionedTestLoc";
```

Comment:
Why did all this end up deleted?

Reply:
Just saw the VersionedTimeseriesControllerIT.

Hard to tell from the github GUI, please confirm this view is just everything getting reset back and the new tests are in VersionedTimeseries...IT.

Reply:
Yes I moved all of my tests to VersionedTimeSeriesIT to separate versioned ts tests from the existing tests